ngraph/core/include/ngraph/opsets/opset4_tbl.hpp

```diff
@@ -155,10 +155,14 @@ NGRAPH_OP(TopK, ngraph::op::v3)
 NGRAPH_OP(Acosh, ngraph::op::v3)
 NGRAPH_OP(Asinh, ngraph::op::v3)
 NGRAPH_OP(Atanh, ngraph::op::v3)
+NGRAPH_OP(GRUSequence, ngraph::op::v4)
+NGRAPH_OP(LSTMSequence, ngraph::op::v0)
 NGRAPH_OP(CTCLoss, ngraph::op::v4)
 NGRAPH_OP(HSwish, ngraph::op::v4)
 NGRAPH_OP(NonMaxSuppression, ngraph::op::v4)
 NGRAPH_OP(Mish, ngraph::op::v4)
+NGRAPH_OP(CTCLoss, ngraph::op::v4)
```

Comment:
CTCLoss is mentioned twice. Please, fix this and make the operations in the alphabet order.